pnpm-lock.yaml

```diff
@@ -1,39 +1,43 @@
 lockfileVersion: '6.0'
 
+settings:
+  autoInstallPeers: true
+  excludeLinksFromLockfile: false
+
 importers:
 
   .:
     devDependencies:
       '@changesets/cli':
         specifier: ^2.26.2
-        version: 2.26.2
+        version: 2.27.5
       '@typespec/prettier-plugin-typespec':
         specifier: ~0.54.0
         version: 0.54.0
       cspell:
         specifier: ^7.3.7
-        version: 7.3.7
+        version: 7.3.9
       eslint:
         specifier: ^8.50.0
-        version: 8.50.0
+        version: 8.57.0
       eslint-plugin-import:
         specifier: ^2.28.1
-        version: 2.28.1(eslint@8.50.0)
+        version: 2.29.1(eslint@8.57.0)
       jest:
         specifier: ^29.7.0
         version: 29.7.0
       prettier:
         specifier: ^3.0.3
-        version: 3.0.3
+        version: 3.2.5
```

Comment:
the format diff could come from prettier version bump?

Reply:
Yes, I think this is the cause.